serf/query.go

```diff
@@ -41,10 +45,12 @@ func (s *Serf) DefaultQueryTimeout() time.Duration {
 
 // DefaultQueryParam is used to return the default query parameters
 func (s *Serf) DefaultQueryParams() *QueryParam {
+	relayFactor := math.Ceil(math.Log10(float64(s.memberlist.NumMembers() + 1)))
```

Comment:
We might want to leave this at zero for now to be conservative for library users and let people opt-in. I think in Consul we could do this for the keyring operations.